### PR TITLE
Fix: GLA02 Campaign Stealth General Building Issues

### PR DESCRIPTION
**Pull request by commy2**
ZH 1.04

- GLA02 Stealth General buildings generally behaves inconsistently when stealthed.
- Some GLA02 Stealth General's Buildings are incompatible with Fortified Structures.
- The GLA02 Stealth General's Buildings are immune to Microwave tanks.
- The GLA02 Stealth General's Barracks, Supply Stash, Black Market has half the health of the regular versions.
- The GLA02 Stealth General's Supply Stash is build in 30 seconds.
- The GLA02 Stealth General's Arms Dealer is build in 50 seconds.
- The GLA02 Stealth General's Black Market is build in 90 seconds.
- The GLA02 Stealth General's Black Market has an un-usable button to upgrade Camo-Netting, despite already being camouflaged.
- The GLA02 Stealth General's Supply Stash is classified as power plant instead of as supply dock.
- The GLA02 Stealth General's Palace does use its day model during the night.

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLABuildings.ini [hunk 1]
```diff
@@ -4207,19 +4330,49 @@ Object GC_Slth_GLAPalace
...
+    ConditionState         = USER_1
+      Model                = UBPalaceEG
+      ParticleSysBone      = Smoke01 SmolderingSmoke
+      Animation            = UBPalaceEG.UBPalaceEG
+      AnimationMode        = LOOP
+    End
+    ConditionState         = USER_1 GARRISONED
+      Model                = UBPalaceEGX
+      ParticleSysBone      = Smoke01 SmolderingSmoke
+       Animation            = UBPalaceEGX.UBPalaceEGX
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLABuildings.ini:4352]: minor indentation error

File: Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLABuildings.ini [hunk 2]
```diff
@@ -4207,19 +4330,49 @@ Object GC_Slth_GLAPalace
...
-    ConditionState         = REALLYDAMAGED GARRISONED
-      Model                = UBPalace_EG
+    ConditionState         = DAMAGED USER_1
+      Model                = UBPalaceEG_D
+      ParticleSysBone      = Smoke01 SmolderingSmoke
+      Animation            = UBPalaceEG_D.UBPalaceEG_D
+      AnimationMode        = LOOP
+    End
+    ConditionState         = DAMAGED USER_1 GARRISONED
+     Model                = UBPalaceEGX_D
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLABuildings.ini:4368]: minor indentation error

File: Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLABuildings.ini [hunk 3]
```diff
@@ -4246,59 +4399,121 @@ Object GC_Slth_GLAPalace
...
     End
-
-
+    ConditionState         = REALLYDAMAGED USER_1 SNOW
+      Model                = UBPalaceEG_ES
+      ParticleSysBone      = Smoke01 SmolderingSmoke
+    End
+    ConditionState         = REALLYDAMAGED USER_1 GARRISONED SNOW
+      Model                = UBPalaceEGX_ES
+     ParticleSysBone      = Smoke01 SmolderingSmoke
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLABuildings.ini:4454]: minor indentation error

**Review by xezon — APPROVED** [review]
Jesus Christ.

**Comment by commy2** [comment]
These are copy pasted from `FactionBuilding.ini`, but sure can fix them if required.

**Comment by xezon** [comment]
Optional.

**Pull request merged**